test/common/json/json_streamer_test.cc

```diff
@@ -131,6 +131,22 @@ TEST_F(JsonStreamerTest, SubMap) {
   EXPECT_EQ(R"EOF({"a":{"one":1,"three.5":3.5}})EOF", buffer_.toString());
 }
 
+class JsonStreamerStringOutputTest : public testing::Test {
+protected:
+  std::string buffer_;
+  Json::StreamerBase<Json::StringOutput> streamer_{buffer_};
+};
+
+TEST_F(JsonStreamerStringOutputTest, Empty) { EXPECT_EQ("", buffer_); }
+
```

Comment:
in my previous implementation of this I used TYPED_TEST so we could check all the methods using both implementations.

Reply:
Let me take a look.

Reply:
This is a great suggestion. Thanks.